backend/routes.js

```
@@ -167,8 +163,6 @@ class RoutesHandler{
 			let rID = request.params.rubricID;
 			qry.getAssignment(rID)
 				.then((result) => {
-					if(result.rowCount === 0) 
-						return response.status(400).json({error: "No assignment under this ID"});
```

This should return an error